benches/needleman-wunsch.rs

```diff
@@ -0,0 +1,104 @@
+use criterion::*;
+
+use clam::distances::alignment_helpers;
+use clam::distances::strings;
+use clam::utils::helpers;
+
+fn needleman_wunsch_dim(c: &mut Criterion) {
+    let mut group = c.benchmark_group("needleman-wunsch-dim");
+    group.significance_level(0.025).sample_size(10);
+
+    let plot_config = PlotConfiguration::default().summary_scale(AxisScale::Logarithmic);
+    group.plot_config(plot_config);
+
+    group.sampling_mode(SamplingMode::Flat);
+
+    for n in [10, 25, 50, 100, 250, 500, 1000] {
+        let data = helpers::gen_data_u8(2, n, 65, 68, 42);
```

Comment:
Use a wider range of minimum and maximum values.

Reply:
Ok-- this was meant to be representative of ACGT sequences which is why I only did a 4 char alphabet